sandbox/tests/requests.rs

```diff
@@ -12,8 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-extern crate exonum;
-extern crate sandbox;
+//extern crate exonum;
```

Comment:
I suppose this file is a stub for additional tests. Personally I prefer to remove it completely and add an issue.